### ISSUE TITLE
Problems of the training curve

### ISSUE THREAD

**Issue by automan000** [AlexHex7/SimGAN_pytorch#2]
Hi, AlexHex7.
I forked your code and did several changes for bug fixing and speed-up.
But my training procedure wasn't pleasant. I tested many hyper-parameters but the results were still terrible. Could you share your training log with me? Thanks a lot.

**Comment by AlexHex7** [comment]
Hi~
In fact, last year when I did this experiments, it was also unstable. 
I have not followed the research area of GAN for a long time. I think you can try yo introduce training techniques from other GAN paper to do this experiments, such as more stable loss function, VAE+GAN ...
